### DISCUSSION TITLE
Add System Tray to All Monitors

### DISCUSSION THREAD

**Discussion by NaeemChakera** [martinet101/ElevenClock#304] [General]
Hi, 
Is it possible if you could replicate the clock effect with the system tray of the Windows 11 taskbar? It would be a great help to people with more than one monitor.
Cheers,
Naeem
![image](https://user-images.githubusercontent.com/50527002/142671331-a2b1b9e4-4196-4d8f-b4e8-d5ae13317bd9.png)

**Comment by marticliment** [comment]
Could try, but that's gonna be hard work... I'll say i'll have that in mind, because ElevenClock is going to be my final project for highschool, and it could be better if I improved the app. but i can't give you no estimated time nor any guarantee.
